Php 55 56 eol (#11421)
newrelic/docs-website: src/content/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements.mdx

  Compatibility note: When PHP 8.0 or 8.1 detects an observability extension, like the New Relic agent, PHP disables Just-In-Time compilation.

  Support for PHP 8.1 does not include Fibers.

>   Support for PHP versions 5.5 and 5.6 will be end of lifed in June 2023.

Reviewer: @mfulb Can we say "deprecated" instead of "end of lifed"?

Author: From the product management point of view "deprecated" means it is a feature that is still in place but could be removed any time in the future.  What we want to convey is support is ending in June.  Perhaps we could say "Support for PHP versions 5.5 and 5.6 will end in June 2023." ?

>   Support for PHP versions 5.5 and 5.6 will be end of lifed in June 2023.

Reviewer: ```suggestion
Support for PHP versions 5.5 and 5.6 ends in June 2023.
```
